libs/cart/src/drivers/magento/queries/update-address.ts

```diff
@@ -9,7 +12,7 @@ import { cartFragment } from './fragments/public_api';
  * This helps us keep query complexity down and save some server CPU cycles.
  * Driver behavior is not guaranteed if Magento no longer processes compound queries in the order they are defined.
  */
-export const updateAddress = gql`
+export const updateAddress = (...extraCartFragments: DocumentNode[]) => gql`
```

Comment:
Why does this have to be a spread argument type?

Reply:
it doesn't have to be. I can change it to `extraCartFragments: DocumentNode[] = []`.

Reply:
I think for that time that's preferable. It makes the `updateAddress` more malleable as it would allow us to add additional args in the future.

Reply:
changed

Reply:
I think he means in the actual query too? I was thinking the same thing

Reply:
oh, nevermind. You did change it there